std/node/module.ts

```diff
@@ -490,6 +490,26 @@ class Module {
     }
   }
 
+  /**
+   * Add the `require` function without the need to creating it, 
+   * that can be used to import CJS modules.
+   * Follows CommonJS resolution similar to that of Node.js,
+   * with `node_modules` lookup and `index.js` lookup support.
+   * Also injects available Node.js builtin module polyfills.
+   *
+   *     const leftPad = require_("left-pad");
+   *     const cjsModule = require_("./cjs_mod");
+   *
+   * @param the module
+   * @return the exports of the required module
+   */
+  static require(path: string): any {
+    var e = new Error();
+    var calledFile = e.stack.split("\n")[2].replace(/^ +at /, '').replace(/:[0-9]+:[0-9]+/,'');
```

Comment:
IMHO this is not a very stable solution.
+ Implementation of `require` might change and introduce extra function layers
+ There is an easy way to break this: nested `require`:

e.g.
`require_mod.ts`:
```ts
import { require } from "./module.ts";
export function require2(path) {
  return require(path);
}
```
`test_require.ts`:
```ts
import { require2 } from "./require_mod.ts";
require2("./tests/cjs/cjs_a");
```

Extracted file path becomes `require2 (file:///Users/kun/Projects/Deno/deno/std/node/require_mod.ts)`, which is incorrect

Reply:
- I can't imagine any change to the implementation that could cause problems.
- How is "nested request" thing handled at node? (i can not test it)

Another drawback may be that "stack" is not standardized, but I don't think it is realistic that it will be removed because it has been around for a long time.

Also I don't know how the performance is affected by the call to "New Error().stack".